package.json

```diff
@@ -27,6 +28,7 @@
     "i18next-browser-languagedetector": "7.2.0",
     "i18next-http-backend": "2.4.3",
     "jszip": "^3.10.1",
+    "lzutf8": "0.6.3",
```

Comment:
👮

> Note: this library is significantly out-of-date and will require a full rewrite to update with recent technologies like JS modules and WebAssembly, and for better compatibility with modern frameworks like Angular and Rect.js. The design and documentation was mostly written in 2014, before any of these were relevant. Unfortunately, it is not maintained anymore, and a rewrite is unlikely to take place in the foreseeable future.

Reply:
Switched to https://www.npmjs.com/package/lz-string